shrine/lib/app.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2018-present the Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+
+import 'backdrop.dart';
+import 'colors.dart';
+import 'home.dart';
+import 'login.dart';
+import 'category_menu_page.dart';
```

Comment:
Alphabetize.

Reply:
Ugh. Could have sworn I optimized all the imports. Thanks for catching it.